x11-wm/pekwm: add 0.3.2

krejvl opened the pull request:
Pekwm lightweight X11 window manager, update to the 0.3.2 version. Tested directly on amd64 and via 32bit chroot on x86. 

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

krejvl commented:
@thesamesam OK then, I don't need to touch it. It works fine, all checks have passed, so we can merge it as it is if possible.

thesamesam commented:
I think it's still not right, I'm afraid. You need to include the test changes that are lost in this ebuild, i.e. if you diff 0.3.0 vs 0.3.2, you lost the test changes because they got added _after_ you made this PR initially.

krejvl commented:
Sorry, my fault, I've misunderstand what you mean by "mandatory". So I've added tests and corrected the dependencies order. I've also tested the changes locally, it seems to work fine. I hope it's fine now.

thesamesam commented:
Sorry, one more nit.. `Signed-off-by` is missing.
